setup.py

```diff
@@ -24,5 +24,5 @@
     author_email='oscm@edx.org',
     license='Apache',
     packages=find_packages(exclude=['*.tests']),
-    install_requires=['slumber', 'PyJWT'],
+    install_requires=['slumber >= 0.7.1', 'PyJWT >= 1.1.0'],
```

Comment:
@dan-f its a fair point, but IMO it's safer and not unduly burdensome to require manual intervention to "opt in" on minor version changes.  Nobody's perfect and some issues introduced in a minor release may not break tests, such as security vulnerabilities.  So, I'm with @clintonb on this.